TrafficCapture/settings.gradle

```diff
@@ -40,3 +40,13 @@ addSubProjects('', new File(rootProject.projectDir,'replayerPlugins'))
 include 'replayerPlugins:jsonMessageTransformers:jsonJoltMessageTransformerProvider:untitled'
 findProject(':replayerPlugins:jsonMessageTransformers:jsonJoltMessageTransformerProvider:untitled')?.name = 'untitled'
 
+// Enable Gradle Scan Publishing for CI/CD
+if (System.getenv().containsKey("CI")) {
```

Comment:
How can I stop getting prompted when I run --scan on my own command line?  Should this CI check be something more specific to our repo?  If a user runs on a shell with CI as a variable, they might not have agreed to the terms.

Reply:
I don't have any objections to make it over the whole repo

Reply:
Can we key this off of an environment variable like "I_AGREE_TO_TERMS_AND_SERVICE_FOR_GRADLE_SCANS"?  That variable wouldn't be accidentally set and is clear.  I'm concerned that if you set it to something generic, or nothing, that somebody will fork the repo, add something that they shouldn't send, and not even realize that they had done so.